indra/llwindow/llwindowwin32.cpp

```diff
@@ -1702,10 +1709,15 @@ void LLWindowWin32::recreateWindow(RECT window_rect, DWORD dw_ex_style, DWORD dw
     auto oldWindowHandle = mWindowHandle;
     auto oldDCHandle = mhDC;
 
+    if (sWindowHandleForMessageBox == mWindowHandle)
+    {
+        sWindowHandleForMessageBox = NULL;
+    }
+
     // zero out mWindowHandle and mhDC before destroying window so window
     // thread falls back to peekmessage
-    mWindowHandle = 0;
-    mhDC = 0;
+    mWindowHandle = NULL;
+    mhDC = NULL;
```

Comment:
Handle isn't a pointer, it contains an integer. Ex: -1 equals invalid, While NULL isn't technically invalid to use, it often implies a pointer, plus Microsoft uses 0. I suggest using Microsoft's standard and leave it as a 0.

Reply:
HWND is a type of pointer to a struct HWND__
![image](https://github.com/user-attachments/assets/b86c3a18-9232-4e66-928e-1dd0d6d7c957)
Semantically it is a pointer, but NULL is defined as 0
Of course, there is almost no difference between 0 and NULL
But if we use NULL everywhere in this class and 0 omly once,
I suggest replace 0 with NULL for better readability

Reply:
Sorry, you are correct.